src/labels/CustomLabels.labels

```diff
@@ -503,6 +503,14 @@ day:      &apos;day&apos;</value>
         <shortDescription>label displayed on PersonalSiteContactInfo if the lookup and email succeed</shortDescription>
         <value>Message sent.  Please check your email for a message that contains a link to your Volunteer information.  Thank you.</value>
     </labels>
+    <labels>
+        <fullName>labelContactLookupAmbiguous</fullName>
+        <categories>PersonalSiteContactInfo</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>label displayed on PersonalSiteContactInfo after lookup is complete</shortDescription>
+        <value>If your record was found an email message that contains a link to your Volunteer information will be sent. Thank you.</value>
```

Comment:
@katherinemitchell - This is the preliminary wording Art and I landed on. I would appreciate your review / input here.